calico/felix/fetcd.py

```diff
@@ -89,15 +92,19 @@ class EtcdAPI(Actor):
     Since the python-etcd API is blocking, we defer API watches to
     a worker greenlet and communicate with it via Events.
 
-    As and when we add status reporting, to avoid needing to interrupt
-    in-progress polls, I expect we'll want a second  worker greenlet that
-    manages an "upstream" connection to etcd.
+    To avoid needing to interupt in-progress polls, another working
+    greenlet is introduced to manage status updating and writing to
+    etcd.
```

Comment:
'... another working greenlet is introduced ...' - remember this docstring needs to continue to make sense in future, so better to say something like '... a different worker greenlet is used ...'